package.json

```diff
@@ -8,7 +8,6 @@
     "/dist"
   ],
   "bin": {
-    "🚘": "./dist/cjs/cli/cli.js",
```

Comment:
Any reason to remove this bin emoji entry?

Reply:
As stated in the commit changing this,
The car is cute and fun but on an operational standpoint it is not usable.
In tty or on terminals not supporting this, it is hard to spot the process and to just enter the process name manually isn't possible this way.
I don't even see the car in htop even though my terminal supports the character which means that it is even incompatible with some applications that may be used on a server

Reply:
There is still the option to use `ipfs-car` command, so this change is out of scope of this PR. You can raise an issue to have it removed and explain the reasoning.

Reply:
I've reverted this change